bindings/java/src/enums.rs

```diff
@@ -0,0 +1,251 @@
+use crate::conversions::f_get_value;
+use crate::primitives::f_i32_from_primitive;
```

Comment:
I think this file could be auto-generated too. My suggested strategy would be:

1. Create a trait:

```rust
trait FromJavaEnum {
     const JAVA_CLASS : &'static str;
     fn from_ordinal(ordinal: i32) -> Option<RustType>;
}
```

2. Auto-generate an impl of this trait for each rust enum when generating the java enum.

3. This file should then just need two (generic) functions: one for T (that returns default if the value is null) and one for Option<T>

Reply:
This was actually an incredible idea, just commited an implem of this, would be very interested in knowing what you do think about it :)